src/components/dataset-viewer/Cell.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts">
+import _ from 'lodash'
+import Vue from 'vue'
+
+import detectType from './detect-type'
+
+const defaultFormatter = (value: any, row: object, column: string): string =>
+  _.isObject(value) ? JSON.stringify(value) : _.toString(value)
+
+const CSS_CLASS = 'dataset-viewer__cell'
+
+type ClassModifierGenerator = (value: any, row: object, column: string, displayedValue: string) => string | undefined
+
+export default Vue.extend({
+  name: 'dataset-viewer-cell',
+
+  props: {
+    column: {},
+    row: {},
+    displayFormatted: {
+      default: false,
+      type: Boolean
+    },
+    formatter: {
+      default: defaultFormatter,
+      type: Function
+    },
+    classModifiers: {
+      default: () => [],
```

Comment:
Could we add some documentation about those props? Especially about class modifiers that seem to be chained?